mappings/net/minecraft/command/argument/DimensionArgumentType.mapping

```diff
@@ -6,7 +6,7 @@ CLASS net/minecraft/unmapped/C_agjrdwvm net/minecraft/command/argument/Dimension
 		ARG 1 context
 		ARG 2 builder
 	METHOD m_nscsmxxv (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
-		ARG 0 id
+		ARG 0 dimension
```

Comment:
I don't like this, what's being passed isn't a dimension object and the fact that it's a dimension id is easily inferred

Reply:
No, it isn't an id either, it's the DimensionArgumentTy- wait, did enigma trick me

Reply:
oh wait am I dumb

Reply:
I think you're right here; that is indeed the dimension being passed

Reply:
no i was wrong, it was the dimension ID; although yeah, this does point to a bad standard going on